lib/phobos_db_checkpoint.rb

```diff
@@ -22,6 +22,10 @@ class << self
     attr_reader :db_config
     attr_accessor :db_config_path, :db_dir, :migration_path
 
+    def table_name_prefix
```

Comment:
Everywhere in this module: http://apidock.com/rails/ActiveRecord/Base/table_name_prefix/class

If we remove this line it doesn't work, models will try to look for table named "failures" and "events" instead of "phobos_db_checkpoint_failures" etc.